dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1221,6 +1221,15 @@ serverCmd()
   SAVE_OPENJ9_JAVA_OPTIONS=${OPENJ9_JAVA_OPTIONS}
   OPENJ9_JAVA_OPTIONS=${SERVER_IBM_JAVA_OPTIONS}
 
+  # Check for any verbose:gc variable set by user
+  # By default we set this to be true unless user specifies otherwise
+  # if [ ! jvmargs, javaoptions, jvmoptionsquoted contains verbose:gc, verbosegc, etc] then SERVER_IBM_JAVA_OPTIONS += -Xverbosegclog:verbosegc.%seq.log,10,1024
+  if [[ $JVM_OPTIONS_QUOTED != *"gc"* ]]
+  then
+    IBM_JAVA_OPTIONS+=" -Xverbosegclog:logs/verbosegc.%seq.log,10,1024"
```

Comment:
One potential issue I can see is that this configures it to write to the logs subdirectory of the current working directory. In principle, I guess it's possible a customer has configured the logs directory to be somewhere else, so you might want to double check what happens in that case and whether the JVM will create the logs subdirectory if it doesn't exist.